local_rl_loop.py

```diff
@@ -79,6 +80,10 @@ def rl_loop():
             holdout_pct=100,
             readouts=10)
 
+        print("See sgf files here?")
+        sgf_listing = subprocess.check_output("ls -l {}/full".format(sgf_dir).split())
```

Comment:
Just use https://docs.python.org/3/library/os.html#os.listdir and filter?

Reply:
see below; i'd like to see these are nonzero size files.

Reply:
https://docs.python.org/3/library/os.html#os.scandir  is the equivalent of `ls -l` in pure python

Reply:
also, `subprocess.X(some_string.split())` is an antipattern. If `some_string` contains an unintended space, it would be the equivalent of passing in a filename to a bash command where the filename had a space in it. In bash, you would quote the filename; the way python would know to quote the filename is by passing in a list as you intended it to be split: `subprocess.check_output(['ls', '-l', filename])`